### PR TITLE
Starting my portfolio

### PR DESCRIPTION
**Pull request by bad6e**
WIP - Portfolio - Bret Doucette - 8.7.15

### CODE REVIEW

File: students/bret_doucette/20150814-bret_doucette-m1.markdown [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+### Giving Feedback
+
+(feedback from me)
+
+### Playing a Part
+
+**@Channel**
+
+Member of @channel - a Turing School Podcast for incoming prospective students. Thus far we have recorded two podcasts. I have also contributed to content ideas as well as taken responsibilities for editing and maintaining the podcast audio.
```
Comment (worace) [inline students/bret_doucette/20150814-bret_doucette-m1.markdown]: this sounds cool. are the episodes up somewhere?

**Comment by worace** [comment]
looks like a solid start. appreciate the thoroughness of the project materials included. would be interested to also see some brief reflections of your own on each project (in addition to the staff / evaluation comments already included). What did you feel went well? What didn't? What techniques or processes did you experiment with? etc.

**Comment by mikedao** [comment]
Good start!

**Comment by bad6e** [comment]
Thanks for the feedback! I adjusted appropriately.

**Pull request merged**